CODING.md

```diff
@@ -24,6 +24,11 @@ See the [node website](https://nodejs.org) to install node and npm.
    * You can then require `src/Main.js` in your code
 * Test and lint changes: see test, lint and test-examples npm script
 
+## Debuging
+* `babel-inline-import-loader` prevents the source map debug in browser.
+   if you want launch server and debug with the original source map, run : `npm run startDebug`.
+* To debug iTowns package on your side project. You can link iTowns package with `npm link ../path/iTowns/` in project folder and auto-transpile when iTowns sources are modified with command `npm run watch` in iTowns folder.
```

Comment:
:+1: however that does not seem to work here: I just run `npm run watch` and modify + save a file in `src`,  but it didn't change anything.

Reply:
could you check in `lib/` folder?

Reply:
It's okay thanks !